vcpkg/commands/env.md

```diff
@@ -0,0 +1,125 @@
+---
+title: vcpkg env
+description: Command line reference for the env command. Creates a clean build environment in your current terminal.
+ms.author: viromer
+ms.date: 7/20/2023
+ms.prod: vcpkg
+---
+
+# vcpkg env
+
+
+## Synopsis
+
+> [!NOTE]
+> This command is only supported on Windows.
+
+```Console
+vcpkg env [<optional command>] [--triplet=<triplet>] [--options]
+```
+
+## Description
+
```

Comment:
I think it could be useful to give a brief explanation of why a user might want to use this command.